src/features/auth/routes/index.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Route, Routes } from 'react-router-dom';
+
+import { Login } from './Login';
+import { Register } from './Register';
+
+export const AuthRoutes = () => {
+  return (
+    <Routes>
+      <Route path="sign-up" element={<Register />} />
```

Comment:
there is a file with all paths defined? in order to import them from there
 ```suggestion
      <Route path={paths.signUp} element={<Register />} />
```

Reply:
Great suggestion, I implemented it with an enum. Thanks!